File: BankingTool.Model/Dto/ResponseDto/UserInitialLoadResponse.cs
namespace BankingTool.Model.Dto.Response
{
    public class UserInitialLoadResponse
    {
        public UserDetailResponse UserDetail { get; set; }
    }
    public class UserDetailResponse
    {
        public int UserId { get; set; }
        public string Password { get; set; }
        public string EmailId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public int City { get; set; }
        public int State { get; set; }
        public int RoleId { get; set; }
    }
}

File: bankingtool.client/src/app/dto/response/user-initial-load-response.ts
export interface UserInitialLoadResponse {
  userDetail: UserDetailResponse;
}

export interface UserDetailResponse {
  userId: number;
  password: string;
  emailId: string;
  firstName: string;
  lastName: string;
  city: number;
  state: number;
  roleId: number;
}
